src/bionics.h

```diff
@@ -79,6 +79,8 @@ struct bionic_data {
     std::map<bodypart_str_id, size_t> cut_protec;
     /**Amount of bullet protection offered by this bionic*/
     std::map<bodypart_str_id, size_t> bullet_protec;
+    /**Amount this modifies dispersion.*/
+    cata::optional<float> dispersion_mod;
```

Comment:
Should fix that compiler warning.
```suggestion
    cata::optional<float> dispersion_mod = cata::nullopt;
```